ibis/backends/postgres/__init__.py

```diff
@@ -179,3 +180,88 @@ def udf(
             name=name,
             language=language,
         )
+
+    def _get_schema_using_query(self, query: str) -> sch.Schema:
```

Comment:
Can you move these utilities to `postgres/datatypes.py`?

Reply:
Not really, this is a method on the backend.

Reply:
Or are you talking about the datatype mapping stuff?

Reply:
Moved the mapping code to `datatypes.py`.